assembly/src/bin/tornado

```diff
@@ -120,7 +120,7 @@ if [[ $JAVA_VERSION == "9.0" || $JAVA_VERSION == "10.0" ]]; then
   exit 0
 fi
 
-TORNADO_FLAGS="-Djava.library.path=${TORNADO_SDK}/lib:$EXTERNAL_DIRS "
+TORNADO_FLAGS="-Djava.library.path=${TORNADO_SDK}/lib"
```

Comment:
Could you please revert to use `$EXTERNAL_DIRS`. We need this for other projects.

Reply:
```suggestion
case "$(uname -s)" in
   CYGWIN*|MINGW32*|MSYS*|MINGW*)
     PATH_SEPARATOR=";"
     ;;

   *)
     PATH_SEPARATOR=":"
     ;;
esac
EXTERNAL_DIRS_P=""
if test "${EXTERNAL_DIRS+defined}"; then
  EXTERNAL_DIRS_P="${PATH_SEPARATOR}${EXTERNAL_DIRS}"
fi 
TORNADO_FLAGS="-Djava.library.path=${TORNADO_SDK}/lib${EXTERNAL_DIRS_P}"
```